src/test/java/com/jcabi/xml/XMLDocumentTest.java

```diff
@@ -349,9 +349,10 @@ public void buildsDomNode() throws Exception {
      */
     @Test
     public void comparesToAnotherDocument() throws Exception {
+        final String xml = "<hi><dude>  </dude></hi>";
         MatcherAssert.assertThat(
-            new XMLDocument("<hi>\n<dude>  </dude></hi>"),
-            Matchers.equalTo(new XMLDocument("<hi><dude>  </dude></hi>"))
+            new XMLDocument(xml),
```

Comment:
@krzyk this test `comparesToAnotherDocument` already had two comparisons, one for equals and one for not equals. All I did was to fix the test because of the `\n`. You think it doesn't make sense to teste equals and not equals in the same test?